### PR TITLE
Improved Sentry error reporting

### PR DESCRIPTION
**Pull request by leo**
This PR:

- ...bumps `@zeit/ncc` to the latest version, so we can enjoy minification
- ...handles global arg parsing errors better (see below)
- ...removes all occurances of `userError` (bad engineering practice)
- ...makes Sentry use a shared utility for reporting errors
- ...prevents errors from being reported while developing

BEFORE

image

AFTER

image

### CODE REVIEW

File: src/commands/upgrade.js [hunk 1]
```diff
@@ -105,7 +105,7 @@ const downgradeToFree = async ({ error }, now) => {
 };
 
 export default async function main(ctx     )                  {
```
Comment (timneutkens) [inline src/commands/upgrade.js]: The whitespace here is *super weird*

Comment (leo) [reply src/commands/upgrade.js]: Fixed!

**Review by timneutkens — COMMENTED** [review]
Does this handle Sentry being down?

**Comment by leo** [comment]
@timneutkens That's handled [here](https://github.com/zeit/now-cli/blob/canary/src/index.js#L579) already.

**Comment by timneutkens** [comment]
Great 👍

**Review by timneutkens — APPROVED** [review]

**Comment by timneutkens** [comment]
Looks fine, even though I don't have all the context of now-cli error handling to correctly judge this.

**Pull request merged**